build.gradle

```diff
@@ -88,8 +88,9 @@ subprojects { subproject ->
 
 	ext {
 		activeMqVersion = '5.15.2'
-		aspectjVersion = '1.8.13'
 		apacheSshdVersion = '1.6.0'
+		aspectjVersion = '1.8.13'
+		assertjVersion = '2.6.0'
```

Comment:
Why do we need to introduce one more testing dependency?

Reply:
Over time, I would like to migrate everything to AssertJ to be consistent with Boot and cloud - I just don't have Phil Webb's diligence to do it all in one go. 

Similar to BDDMockito migration.

It's becoming a bit painful to use different libraries in the various projects I contribute to.

If you don't agree, I will reconsider.

Reply:
Yes, I agree about consistency and I definitely like AssertJ with all its smart API.

Only the problem that we even have failed for several years already to update our imports order to align at least with Spring Framework. And also we have failed the same way to turn one all the Checkstyle rules.

So, that's how I'm afraid we will fail with AssertJ as well.

As you said: too much various projects we contribute to...

But at least we can start to use AssertJ in new tests! 😄 
OK, let's see how is it going to work!

Reply:
I think at one point I wrote a [Rewrite](https://github.com/Netflix-Skunkworks/rewrite) rule to automatically convert JUnit style asserts to AssertJ, if that's interesting to you.

Reply:
@jkschneider I'd certainly be interested in taking a look, but I'd probably start with a small project than SI, though 😄